### DISCUSSION TITLE
Allow users to operate the code area, display in other languages, and change image resolution

### DISCUSSION THREAD

**Discussion by zhraa11** [sbrin/lopaka#163] [Ideas]
Hello, during the use, I discovered some functions that can be quickly integrated and greatly improve the user experience.
First, it allows users to adjust the resolution of uploaded images to fit low-resolution screens.
Second, for non-English users, allow them to enter their own language, and take a model of it and insert it in the form of a picture.
Third, allow users to perform operations in the code area, such as pasting, deleting, etc.
Thank you for reading my question and your help for the MCU users.

**Comment by sbrin** [comment]
@zhraa11 thank you for your feedback

> "and take a model of it and insert it in the form of a picture"

I can't understand what model do you mean. Could you please describe this process with more details?

**Comment by zhraa11** [comment]
![image](https://github.com/sbrin/lopaka/assets/98087300/01768157-1200-40c1-87b0-e0a548af2d45)

Like this, I can input text in other languages, generate image data, and finally insert it into the code in the form of images.

**Comment by sbrin** [comment]
Regarding image import options: we have a work in progress here https://image-import-wizard.lopaka-dev.pages.dev/
@zhraa11 please tell what you think about it:
<img width="656" alt="Screenshot 2024-05-04 at 20 19 59" src="https://github.com/sbrin/lopaka/assets/1325864/01ddf6ee-1b77-41ae-8d3f-aca457932abf">

(probably it will be changed in future)

**Comment by zhraa11** [comment]
This is amazing.I think this section is pretty polished, but I think zoom would be better if it allowed users to enter any number they wanted.
